templates/publisher/publicise/github_badges.html

```diff
@@ -49,12 +49,7 @@ <h4>Promote your snap using embeddable GitHub badge</h4>
         <label>HTML:</label>
       </div>
       <div class="col-7">
-        <div class="p-code-copyable not-cli">
-          <code class="p-code-copyable__input" id="snippet-badge-html">&lt;a href="https://snapcraft.io/{{ snap_name }}"&gt;
-    &lt;img alt="{{ snap_title }}" src="https://snapcraft.io/{{ snap_name }}/badge.svg" /&gt;
-  &lt;/a&gt;</code>
-          <button class="p-code-copyable__action js-clipboard-copy" data-clipboard-target="#snippet-badge-html">Copy to clipboard</button>
-        </div>
+        <pre><code id="snippet-badge-html">&lt;a href="https://snapcraft.io/{{ snap_name }}"&gt;&lt;img alt="{{ snap_title }}" src="https://snapcraft.io/{{ snap_name }}/badge.svg" /&gt;&lt;/a&gt;</code></pre>
```

Comment:
The `id` again, I won't repeat this comment, but you can review other code blocks as well and get rid of ids if they were only used for copy button.

Reply:
needed to update the content of the code snippet when you change the badge content